TESTS/network/wifi/README.md

```diff
@@ -58,6 +59,8 @@ Please refer to the following table for priorities of test cases. Priorities are
 |     |                                         | NSAPI_SECURITY_WPA         | SHOULD   |
 |     |                                         | NSAPI_SECURITY_WPA2        | SHOULD   |
 |     |                                         | NSAPI_SECURITY_WPA_WPA2    | MUST     |
+|     |                                         | NSAPI_SECURITY_WPA3_WPA2   | SHOULD   |
```

Comment:
I would prefer NSAPI_SECURITY_WPA2_WPA3 for consistency. But this is just my opinion and can be ignored if no one else feels the same.